docs/subnets/customize-a-subnet.md

```diff
@@ -373,6 +373,71 @@ interface NativeMinterInterface {
 
 _Note: Both `ContractDeployerAllowList` and `ContractNativeMinter` can be used together._
 
+### Configuring Dynamic Fees
+
+You can configure the parameters of the dynamic fee algorithm on chain using the `FeeConfigManager`. In order to activate this feature, you will need to provide the `FeeConfigManager` in the genesis:
+
+```json
+{
+  "config": {
+    "feeManagerConfig": {
+      "blockTimestamp": 0,
+      "adminAddresses": ["0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC"]
+    }
+  }
+}
+```
+
+The FeeConfigManager implements the FeeManager interface which includes the same AllowList interface used by ContractNativeMinter, TxAllowList, etc. To see an example of the AllowList interface, see the [TxAllowList](#restricting-who-can-submit-transactions) above.
+
+The `Stateful Precompile` powering the `FeeConfigManager` adheres to the following Solidity interface at `0x0200000000000000000000000000000000000003` (you can load this interface and interact directly in Remix):
+
+The FeeConfigManager implements the following interface:
```

Comment:
Both paragraphs above end with :, please refactor and remove one of them.

Reply:
Nice catch, just updated